test/box-luatest/gh-5602-environment-vars-cfg_test.lua

```diff
@@ -0,0 +1,198 @@
+local t = require('luatest')
+local fio = require('fio')
+
+local justrun = require('luatest.justrun')
+local treegen = require('luatest.treegen')
+
+local g = t.group()
+
+local tmpdir = fio.tempdir()
```

Comment:
It is not deleted. I would re-use treegen's directory here.

Reply:
Fixed:

```diff
diff --git a/test/box-luatest/gh-5602-environment-vars-cfg_test.lua b/test/box-luatest/gh-5602-environment-vars-cfg_test.lua
index d7ffca2fb7..41a6781185 100644
--- a/test/box-luatest/gh-5602-environment-vars-cfg_test.lua
+++ b/test/box-luatest/gh-5602-environment-vars-cfg_test.lua
@@ -6,13 +6,10 @@ local treegen = require('luatest.treegen')
 
 local g = t.group()
 
-local tmpdir = fio.tempdir()
-local listen_sock1 = fio.pathjoin(tmpdir, 'listen1.sock')
-local listen_sock2 = fio.pathjoin(tmpdir, 'listen2.sock')
-
 -- Check that environment cfg values are set correctly.
 g.test_gh_5602_tarantool_env_variables_1 = function()
     local dir = treegen.prepare_directory({}, {})
+    local listen_sock = fio.pathjoin(dir, 'listen.sock')
     local script_name = 'test_gh_5602_tarantool_env_variables_1.lua'
     local script_body = ([[
 local test = require('tap').test('gh-5602')
@@ -26,11 +23,11 @@ test:is(box.cfg['strip_core'], false, 'strip_core')
 test:is(box.cfg['log_format'], 'json', 'log_format is not set')
 test:is(box.cfg['log_nonblock'], false, 'log_nonblock')
 
-os.exit(test:check() and 0 or 1, true) ]]):format(listen_sock1)
+os.exit(test:check() and 0 or 1, true) ]]):format(listen_sock)
     treegen.write_file(dir, script_name, script_body)
     local opts = { nojson = true,  stderr = true }
     local env = {
-        TT_LISTEN = listen_sock1,
+        TT_LISTEN = listen_sock,
         TT_READAHEAD = 10000,
         TT_STRIP_CORE = false,
         TT_LOG_FORMAT = 'json',
@@ -53,6 +50,8 @@ end
 -- Check that environment cfg values are set correctly.
 g.test_gh_5602_tarantool_env_variables_2 = function()
     local dir = treegen.prepare_directory({}, {})
+    local listen_sock1 = fio.pathjoin(dir, 'listen1.sock')
+    local listen_sock2 = fio.pathjoin(dir, 'listen2.sock')
     local script_name = 'test_gh_5602_tarantool_env_variables_2.lua'
     local script_body = ([[
 local test = require('tap').test('gh-5602')

```